src/components/Image/Image.ts

```diff
@@ -59,7 +73,8 @@ export class Image extends NostoElement {
     layout: String,
     crop: String,
     alt: String,
-    sizes: String
+    sizes: String,
+    breakpoints: JSON
```

Comment:
Do we need a JSON type here? This unnecessarily exposes other methods that are part of JSON to breakpoints field. Should we use a property decorator here with type here as String?

Reply:
we don't need it it, but String would mean that we need custom treatment of the breakpoints file and need to do the JSON parsing before passing it to the unpic transform pipeline